libs/fetcher.js

```diff
@@ -335,7 +335,14 @@ Fetcher.middleware = function (options) {
                     }
                     if (err) {
                         var errResponse = getErrorResponse(err);
-                        res.status(errResponse.statusCode).json(responseFormatter(req, res, errResponse.output));
+                        if (req.query && req.query.returnMeta) {
```

Comment:
Yeah, we were doing this for normal requests but not for error requests so its kind of a bug.